lis/core/LIS_readConfig.F90

```diff
@@ -791,12 +791,12 @@ subroutine LIS_readConfig()
   
   call LIS_parseTimeString(time,LIS_rc%pertrestartInterval)
 
-  LIS_rc%pert_bias_corr = 1
```

Comment:
Note that the current master from NASA that I pulled in December had the bias correction by default turned 'ON' ignoring any related user setting in lis.config